lib/aur-chroot

```diff
@@ -260,12 +304,34 @@ if (( print_path )); then
 fi
 
 if (( build )); then
-    # use makechrootpkg -c as default build command (sync /root container)
-    # arguments after -- are used as makechrootpkg arguments
-    # shellcheck disable=SC2086
-    $AUR_PACMAN_AUTH "${auth_args[@]}" makechrootpkg -r "$directory" \
-         "${bindmounts_ro[@]/#/-D}" "${bindmounts_rw[@]/#/-d}" \
-         "${makechrootpkg_args[@]}" -- "${makechrootpkg_makepkg_args[@]}"
+    bindmounts_rw+=(
+        "$PWD"
+        "${PKGDEST:-$PWD}:/pkgdest"
+        "${LOGDEST:-$PWD}:/logdest"
+        "${SRCDEST:-$PWD}:/srcdest"
+        "${SRCPKGDEST:-$PWD}:/srcpkgdest"
+    )
+
+    # XXX: if you change this it doesn't reflect modifications
+    printf "u %q $(id -u) \"makepkg user\" /build /bin/bash\n" "$makepkg_user" | \
+        $AUR_PACMAN_AUTH systemd-sysusers --root "$directory"/root --replace=/usr/lib/sysusers.d/build.conf -
+
+    # XXX: avoid recloning each time?
+    $AUR_PACMAN_AUTH systemd-nspawn \
+        -D "$directory"/root \
+        --chdir="$PWD" \
+        --user="$makepkg_user" \
+        --volatile=overlay \
+        --setenv=PKGDEST=/pkgdest \
+        --setenv=LOGDEST=/logdest \
+        --setenv=SRCDEST=/srcdest \
+        --setenv=SRCPKGDEST=/srcpkgdest \
+        --setenv=BUILDDIR="$PWD" \
+        --setenv=MAKEPKG_CONF="$makepkg_conf" \
+        --setenv=SOURCE_DATE_EPOCH="${SOURCE_DATE_EPOCH:-$(date +%s)}" \
+        "${bindmounts_ro[@]/#/--bind-ro=}" \
+        "${bindmounts_rw[@]/#/--bind=}" \
+        makepkg --cleanbuild --force --syncdeps --noconfirm "${makepkg_args[@]}" -- "$@"
```

Comment:
I wouldn't hardcode any makepkg arguments here. It's worth considering to even leave out the `makepkg` command entirely, so that people could use `aurutils` installed inside the chroot, e.g. `aur chroot aur build ...`

Reply:
Agree about the makepkg arguments. About removing makepkg I'm a bit concerned: wouldn't this radically change the semantics of `aur chroot` (and make a bunch of flags useless?), are you okay with that?

Reply:
Yeah, maybe changing the semantics could be left for a future PR. For syncdeps, aur-build also does not include those flags by default. They could propagated if say, the user does `aur-build --syncdeps`